api-reference/docstubs/api/calendar-allowedcalendarsharingroles.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "allowedCalendarSharingRoles"
```

Comment:
For functions and actions, we use the convention "<binding type>: <name". So this title should be "calendar: allowedCalendarSharingRoles".

Reply:
+1 - this seems to be busted across the board.

Reply:
+1.